### ISSUE TITLE
[功能请求]希望作者添加对a1111-sd-webui-tagcomplete插件的支持

### ISSUE THREAD

**Issue by aylz10** [Physton/sd-webui-prompt-all-in-one-app#1]
希望作者添加对a1111-sd-webui-tagcomplete插件的支持，prompt-all-in-one和tagcomplete插件一起使用才是完全体

**Comment by Physton** [comment]
对 a1111-sd-webui-tagcomplete 插件支持。相当于把 stable diffusion webui 扩展的那一套功能完全重写了，暂不考虑。

**Comment by xhoxye** [comment]
> 对 a1111-sd-webui-tagcomplete 插件支持。相当于把 stable diffusion webui 扩展的那一套功能完全重写了，暂不考虑。

刚找到这里，还没安装，不知道是否像a1111插件一样支持优先读取本地翻译词典。
还有能否插入到fooocus里？
